scalameter-core/src/main/scala/org/scalameter/keys.scala

```diff
@@ -4,10 +4,12 @@ package org.scalameter
 
 import java.util.Date
 import scala.collection._
+import org.scalameter.picklers.Pickler
+import org.scalameter.picklers.Implicits._
```

Comment:
Please sort these alphabetically (would be good to have a linter that warns about this, but we don't have a setup for that yet).